README.md

```diff
@@ -80,6 +80,18 @@ Documentation
 
 [docs](docs/) directory contains documents about designs and specifications.
 
+Limitations
+-----------
+
+The following webhooks do not work.
+- Validating Webhook to deny an ephemeral container from running as root user.
+- Mutating Webhook to force an ephemeral container to run as non-root user.
+
+This is because `kubectl debug` command can not specify a user to run ephemeral containers.
```

Comment:
It would be good to clarify that the restrictions are intentional.

```suggestion
The behavior of the webhooks are restricted intentionally as follows:
- Validating Webhook does not deny an ephemeral container from running as root user.
- Mutating Webhook does not force an ephemeral container to run as non-root user.

This is because `kubectl debug` command cannot specify a user to run ephemeral containers.
```